.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
do we push any of these files currently? If so we might need to remove them from the remote

Reply:
By mistake, removed.